### PR TITLE
fix : expect a sdkName based on the test platform

### PR DESCRIPTION
**Pull request by rxlabz**
## :loudspeaker: Type of change

- [x] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring

## :scroll: Description

- define the sdkname based on the platform in the test
- fix the GitHub action to run `pub run test .` instead of `pub run test test/*`

## :bulb: Motivation and Context

- #103 broke a test based on the sdkName

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ jobs:
         - uses: actions/checkout@v2
         - run: pub get
         - name: Test on Chrome
-          run: pub run test -p chrome test/*
+          run: pub run test -p chrome .
```
Comment (rxlabz) [inline .github/workflows/dart.yml]: when we run on `test/*` Dart try to run test on each file in the folder, but test_utils.dart is not a test file.

Comment (marandaneto) [inline .github/workflows/dart.yml]: don't we have tests only on https://github.com/getsentry/sentry-dart/tree/main/dart/test ?

Comment (rxlabz) [reply .github/workflows/dart.yml]: https://github.com/getsentry/sentry-dart/pull/112#discussion_r504547793 :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -17,6 +17,7 @@
 - new Dart code file structure #96 
 - Base the sdk name on the platform (`sentry.dart` for io & flutter, `sentry.dart.browser` in a browser context) #103 
 - Single changelog and readme for both packages #105
+- fix a test regression introduced by #103 #105
```
Comment (marandaneto) [inline CHANGELOG.md]: I'd suggest keeping the PR's title and changelog aligned.

if I read the PR's title, it doesn't look like it's fixing a test regression.

Comment (marandaneto) [reply CHANGELOG.md]: ok now I see it: based on the **test platform**

File: .github/workflows/dart.yml (hunk 1 above)
Comment (rxlabz) [reply .github/workflows/dart.yml]: I changed it to `pub run test -chrome test`

**Review by marandaneto — APPROVED** [review]
LGTM

**Pull request merged**